src/features/client/components/RestaurantInspectCard/index.jsx

```diff
@@ -78,12 +79,12 @@ function RestaurantInspectCard({ deal, cartOpened }) {
               </Text>
             )}
             <Text fontSize="14px" wordBreak="break-word" color="lightslategray">
-              {deal.recipeBio}
+              {deal.bio}
             </Text>
           </Box>
           <Flex alignItems="center">
             <Text fontSize="16px" paddingRight="20px">
-              €{deal.recipePrice}
+              {deal.price}€
```

Comment:
`{deal.price} €`

Reply:
Wraps on display, if space present. Can try css magic for it tho

Reply:
lets leave how it is then